oci/table_oci_autoscaling_auto_scaling_policy.go

```diff
@@ -0,0 +1,248 @@
+			},
+			{
+				Name:        "id",
+				Description: "The ID of the autoscaling policy that is assigned after creation.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "auto_scaling_configuration_id",
+				Description: "The OCID of the autoscaling configuration.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "is_enabled",
+				Description: "Whether the autoscaling policy is enabled.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "policy_type",
+				Description: "The type of autoscaling policy.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "Time that autoscaling policy was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated.Time"),
+			},
+			{
+				Name:        "capacity",
+				Description: "The capacity requirements of the autoscaling policy.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAutoscalingAutoScalingPolicy,
+			},
+
+			// Standard Steampipe columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+
+			// Standard OCI columns
+			{
+				Name:        "compartment_id",
+				Description: ColumnDescriptionCompartment,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CompartmentId"),
+			},
+			{
+				Name:        "tenant_id",
+				Description: ColumnDescriptionTenantId,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     plugin.HydrateFunc(getTenantId).WithCache(),
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+type autoscalingpolicyInfo struct {
+	Id                         *string
+	DisplayName                *string
+	TimeCreated                *common.SDKTime
+	Capacity                   *autoscaling.Capacity
+	IsEnabled                  *bool
+	PolicyType                 *string
+	AutoScalingConfigurationId *string
+	CompartmentId              *string
+}
+
+//// LIST FUNCTION
+
+func listAutoscalingAutoScalingPolicies(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	configuration := h.Item.(autoscaling.AutoScalingConfigurationSummary)
+	region := d.EqualsQualString(matrixKeyRegion)
+	compartment := d.EqualsQualString(matrixKeyCompartment)
+	logger.Debug("listAutoscalingAutoScalingPolicies", "OCI_REGION", region)
+
+	equalQuals := d.EqualsQuals
+	// Create Session
+	session, err := autoScalingService(ctx, d, region)
+	if err != nil {
+		logger.Error("oci_autoscaling_auto_scaling_policy.listAutoscalingAutoScalingPolicies", "connection_error", err)
+		return nil, err
+	}
+
+	//Build request parameters
+	request := buildAutoscalingAutoScalingPolicyFilters(equalQuals)
+	request.Limit = types.Int(100)
+	request.AutoScalingConfigurationId = configuration.Id
```

Comment:
@ParthaI did we miss adding a check for `auto_scaling_configuration_id` optional qual?

Reply:
Added a check for it.

https://github.com/turbot/steampipe-plugin-oci/pull/513/files#diff-e2964d821977ddd037fda883617dadd4ab9e92f5716f0571881f14f2209a7da7R127